lib/ansible/modules/cloud/amazon/ec2_elb.py

```diff
@@ -212,9 +212,9 @@ def _await_elb_instance_state(self, lb, awaited_state, initial_state, timeout):
             elif self._is_instance_state_pending(instance_state):
                 # If it's pending, we'll skip further checks and continue waiting
                 pass
-            elif (awaited_state == 'InService'
-                  and instance_state.reason_code == "Instance"
-                  and time.time() >= wait_timeout):
+            elif (awaited_state == 'InService' and
+                  instance_state.reason_code == "Instance" and
+                  time.time() >= wait_timeout):
```

Comment:
PEP8 specifies the opposite.  Break before binary operators: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator